core/integration/client_tests.go

```diff
@@ -523,8 +519,6 @@ func (env *Env) setupHistory(ctx context.Context, directory *pb.Directory, userI
 			}
 			if _, err := env.Sequencer.RunBatch(ctx, &spb.RunBatchRequest{
```

Comment:
And here and the other places in this file.  Maybe a helper function is needed

Reply:
Done, a helper was added in #1341.